src/VisualStudio/CSharp/Impl/Options/AdvancedOptionPageControl.xaml

```diff
@@ -19,6 +19,8 @@
                               Content="{x:Static local:AdvancedOptionPageStrings.Option_Enable_full_solution_analysis}" />
                     <CheckBox x:Name="Perform_editor_feature_analysis_in_external_process"
                               Content="{x:Static local:AdvancedOptionPageStrings.Option_Perform_editor_feature_analysis_in_external_process}" />
+                    <CheckBox x:Name="Enable_navigation_to_decompiled_sources"
```

Comment:
Lack of a checkbox in in the VB property page is intentional?

Reply:
💭 The output from the decompiler is always C# right now. Also the option isn't language-specific so selecting it here works for VB sources.

Reply:
This still seems like a language specific option.  i.e. someone might not want to go to C# decompiled sources if they're in VB, but might want it from C#.  

that said, i think it's fine to make this a 'global' option that applies to all languages.